### ISSUE TITLE
Display system/base tags when viewing metadata

### ISSUE THREAD

**Issue by pmeijer** [webgme/webgme-taxonomy#558]
These should only be displayed when view the metadata. 
![image](https://github.com/user-attachments/assets/5d6143c0-996d-45b8-b716-b1cd57d297e7)

**Comment by romancow** [comment]
Could you expand on this a little bit? Do you mean you don't want it to show the "Base" terms when you click an upload's "info" button? What it showed should be based on what's being passed from the backend.

Also, this looks like it's not using the json schema form, which it should be now.

**Comment by pmeijer** [comment]
The screen shot is from the old information view. 
I realize my description was pretty bad.. I meant that the system/base tags are no longer displayed when bringing up the information (read-only) modal and that they should be. 
The part about not showing it refers to when you upload a new version of the tags. In that case those tags will be regenerated (automatically) for the new version.

**Comment by romancow** [comment]
So I looked into this. The issue is that the previous version (one in the screenshot) is based solely on the tag values. Basically there is no schema, it just showing whatever raw tag values it gets.

The JSON schema form works based on an actual json schema. This is what it uses to construct the from. The tag values are then what it uses for the initial values to populate this form. So why this new form isn't showing the "Base" vocabulary is because it's in the tag values, but not in the actual json schema.

I'm not sure why the Base vocabulary isn't in the json schema. It actually is in my local dev environment, but I can't tell what the difference is there that would make it show up in one place and not another. I don't know, let me know @pmeijer  if you have any ideas on why the Base vocabulary wouldn't be showing up in the json schema.

**Comment by pmeijer** [comment]
I see what you mean. Although I can't find exactly where this is happening I think it's not part of the schema since the Base terms are system terms and read-only.
We can discuss how to treat these, no need to jump on anything at this point IMO.

**Comment by romancow** [comment]
Yeah, maybe a solution would be a different url or parameter to get the schema with the system/read-only terms
